src/specs/api.yaml

```diff
@@ -458,16 +458,11 @@ components:
           type: string
           example: GetEmbedding
           description: The name of this work -- must be `GetEmbedding`.
```

Comment:
I see that there is no schema defined for the work response for GetEmbedding -- can you add one too? This will make our job easier in the future when we decide to enable the validation of responses received from the worker.